content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/02-schema-incompatibilities.mdx

```diff
@@ -52,7 +52,7 @@ model Post {
 }
 ```
 
-Because the `DEFAULT` constraint has not been added to the database when mapping the Prisma 1 datamodel to the database with `prisma deploy`, Prisma 2 doesn't recognize it during introspection.
+Because the `DEFAULT` constraint has not been added to the database when mapping the Prisma 1 datamodel to the database with `prisma1 deploy`, Prisma 2 doesn't recognize it during introspection.
```

Comment:
```suggestion
Because the `DEFAULT` constraint has not been added to the database when mapping the Prisma 1 datamodel to the database with `prisma deploy`, Prisma 2 doesn't recognize it during introspection.
```